listenbrainz/webserver/timescale_connection.py

```diff
@@ -6,6 +6,10 @@
 
 def init_timescale_connection(logger, conf):
     global _ts
+
+    if not conf["SQLALCHEMY_TIMESCALE_URI"]:
+        return None
```

Comment:
None not needed, can just be `return`. not important for this release

Reply:
Fixed.